compiler/rustc_ast_passes/src/errors.rs

```diff
@@ -823,3 +823,10 @@ pub(crate) struct DuplicatePreciseCapturing {
     #[label]
     pub bound2: Span,
 }
+
+#[derive(Diagnostic)]
+#[diag(ast_passes_missing_extern_abi)]
+pub(crate) struct MissingExternABI {
```

Comment:
```suggestion
pub(crate) struct MissingExternAbi {
```

This is how we write it everywhere else.